packages/manager/modules/pci/src/projects/project/ai-dashboard/components/ai-pipeline/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,12 @@
+{
+  "pci_ai_dashboard_pipeline_stock": "1. Stocker",
```

Comment:
**nitpick:** Hardcoding of `1.` can be sent from HTML to avoid static content in translations.